client/src/components/Profile.js

```diff
@@ -0,0 +1,19 @@
+import Navigation from "./Navigation";
+
+export default function Profile() {
+  return (
+    <div>
+      <header className="header">
+        <h1>
+          <span className="cover-title">Profile</span>
```

Comment:
a span inside a h1 seems counter-intuitive unless you want to treat different parts of the h1 differently, but since you put all of if inside the span, it is unnecessary.
You can just put the className on the h1 instead and get rid of the span